### ISSUE TITLE
Pa11y and possible other dependency issues

### ISSUE THREAD

**Issue by shumatepf** [cloud-gov/pages-uswds-jekyll#289]
The GitHub Action task for running pa11y is consistently failing. When upgrading pa11y to 3.0.1, the action passes but the tests still fail. I noticed this when the [action triggered from dependabot](https://github.com/shumatepf/uswds-site-sample/actions/runs/3735389750/jobs/6352134807) bumping the pa11y version succeeded. Below is the task output for pa11y from that action:

```
Run npm run pa11y-ci:sitemap

> federalist-uswds-jekyll@1.3.0 pa11y-ci:sitemap /home/runner/work/uswds-site-sample/uswds-site-sample
> pa11y-ci --sitemap http://localhost:4000/sitemap.xml --sitemap-exclude "/*.pdf"  --sitemap-exclude "/admin"

Running Pa11y on 11 URLs:
 > http://localhost:4000/collections/ - Failed to run
 > http://localhost:4000/document-with-sidenav/ - Failed to run
 > http://localhost:4000/document/ - Failed to run
 > http://localhost:4000/201[9](https://github.com/shumatepf/uswds-site-sample/actions/runs/3735389750/jobs/6352134807#step:8:10)/01/[13](https://github.com/shumatepf/uswds-site-sample/actions/runs/3735389750/jobs/6352134807#step:8:14)/an-extra/ - Failed to run
```

This can be reproduced on any copy of the `pages-uswds-jekyll` template, including the one linked from the above action.

**Comment by shumatepf** [comment]
@drewbo The [action ran initially](https://github.com/shumatepf/uswds-site-sample/actions/runs/3735385307) when creating a clone of the template failed with the same issue.

**Comment by shumatepf** [comment]
pa11y/pa11y-ci#198 actually addresses this issue

**Comment by drewbo** [comment]
Ah, apologies I only checked the latest builds. Good research and we can downgrade the prescribed runner in the template until this is resolved. Thanks!

**Comment by drewbo** [comment]
Closed via #291
